sdk/tables/Azure.Data.Tables/src/Generated/ServiceRestClient.cs

```diff
@@ -40,6 +41,7 @@ public ServiceRestClient(ClientDiagnostics clientDiagnostics, HttpPipeline pipel
             }
 
             this.url = url;
+            this.secondaryUrl = url.Insert(url.IndexOf('.'), "-secondary");
```

Comment:
At some point we may want to consider implementing a way to parse the full connection string like storage does [here](https://github.com/Azure/azure-sdk-for-net/blob/3065b59a09795d9bf14064c921e80784d34bd63e/sdk/storage/Azure.Storage.Common/src/Shared/StorageConnectionString.cs#L239)